src/modules/refresh-list.js

```diff
@@ -0,0 +1,14 @@
+import { DisplayList } from "../index"
+
+const refreshBtn = document.querySelector('.fa-arrows-rotate');
+
+
+function refresh () {
```

Comment:
kindly. In this line, I recommend using the function Arrow. This helps you to have your code more tidy and short.